assets/js/nav-scroll.js

```diff
@@ -0,0 +1,12 @@
+let siteNav = document.querySelector('.site-nav');
+
+document.addEventListener('DOMContentLoaded', () => {
+    const scroll = localStorage.getItem('scroll');
+    if (scroll !== null) {
+        siteNav.scrollTop = parseInt(scroll);
+    }
+});
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('scroll', siteNav.scrollTop);
+});
```

Comment:
Unnatural; this is changing the behavior users are used to and expect.

Reply:
This was added because currently the users lose their place in the left navigation when clicking on a page because the new page scrolls right to the top.